tei/la-aurora-en-copacabana.xml

```diff
@@ -1352,7 +1352,6 @@
           </sp>
         </div>
         <div type="scene" resp="#rojas" n="9">
```

Comment:
Turns out I did not actually delete it, but I think this scene div is pointless. It isonly in there due to the lion and tiger entering the stage, but they do not speak anyway and thus won't appear in any metric.